libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest.m

```diff
@@ -281,4 +274,18 @@ + (SFRestMethod)sfRestMethodFromHTTPMethod:(NSString *)httpMethod {
     return restMethodName;
 }
 
++ (NSString *)toQueryString:(NSDictionary *)components {
+    NSMutableString *params = [NSMutableString new];
+    if (components) {
+        [params appendString:@"?"];
+        NSUInteger countParams = [components count];
+        for (NSString *paramName in [components allKeys]) {
+            [params appendString:paramName];
+            [params appendString:@"="];
+            [params appendString:[components[paramName] stringByURLEncoding]];
+            if (countParams > 1) [params appendString:@"&"];
+        }
```

Comment:
I think you can remove the `if` block here and just remove the last `&` after the `for` block.